fhir-persistence-schema/src/test/resources/test/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Blocks the Addition of the test files into git
+**/*.key.nowrite
```

Comment:
why does it end in `.nowrite`?
I think the pattern should match the name given in the examples in the README:  `--tenant-key-file tenant.key.file`

Reply:
right... I chose to ignore the files (.nowrite), as in one of the tests, I actually write the file. I wanted to not block the rest of the test files.  in src/test/resources/test/

Reply:
ah, ok.  i guess i was thinking this was a higher-level directory and we were trying to help prevent people from accidentally checking in their keys.  now i understand its just for the test.